pkg/sloop/server/internal/config/config.go

```diff
@@ -33,6 +33,7 @@ type SloopConfig struct {
 	DisableKubeWatcher       bool          `json:"disableKubeWatch"`
 	KubeWatchResyncInterval  time.Duration `json:"kubeWatchResyncInterval"`
 	WebFilesPath             string        `json:"webfilesPath"`
+	BindAddress              string        `json:"bindIp"`
```

Comment:
nit: can you please make sure struct field name and json property name are in sync?

Reply:
good call, will do